static/js/dashboard-charts.js

```diff
@@ -436,65 +436,62 @@ function createColorsArray() {
     return colorArray;
 }
 
-function renderLineChart(seriesArray, metricsDatasets, labels, panelId, chartType, flag) {
-	const colors = createColorsArray();
-    let gridLineColor;
-    let tickColor;
+function renderLineChart(seriesData, panelId) {
+	let classic = ["#a3cafd", "#5795e4", "#d7c3fa", "#7462d8", "#f7d048", "#fbf09e"]
```

Comment:
do we want ot use names here ?

Reply:
We will have the same color for the lines in both modes (light and dark), so I think this will be fine.